<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManualControlDialogVFF</class>
 <widget class="QDialog" name="ManualControlDialogVFF">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manual System Control</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_10">
         <property name="title">
          <string>Signal Generation</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Source</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QCheckBox" name="SourceCE">
                 <property name="text">
                  <string>Chip Enable</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="SourceRFEN">
                 <property name="text">
                  <string>RF Enable</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="SourceLocked">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Locked</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Power:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="SourcePower">
                   <item>
                    <property name="text">
                     <string>-1dBm</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>+1dBm</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>+2.5dBm</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>+3.5dBm</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>+4.5dBm</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>+5.5dBm</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>+6.5dBm</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>+7dBm</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>Frequency:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="SIUnitEdit" name="SourceFrequency"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12">
            <item>
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>Attenuator</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QDoubleSpinBox" name="Attenuator">
                 <property name="suffix">
                  <string>db</string>
                 </property>
                 <property name="minimum">
                  <double>-31.750000000000000</double>
                 </property>
                 <property name="maximum">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.250000000000000</double>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_5">
              <property name="title">
               <string>Amplifier</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <item>
                <widget class="QCheckBox" name="SourceAmplifierEnable">
                 <property name="text">
                  <string>Enable</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_11">
         <property name="title">
          <string>Signal Analysis</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QGroupBox" name="groupBox_7">
            <property name="title">
             <string>LO</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QCheckBox" name="LOCE">
                 <property name="text">
                  <string>Chip Enable</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="LORFEN">
                 <property name="text">
                  <string>RF Enable</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="LOlocked">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="text">
                  <string>Locked</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QFormLayout" name="formLayout_3">
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_8">
                   <property name="text">
                    <string>Freq. Type:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QComboBox" name="LOFreqType">
                   <item>
                    <property name="text">
                     <string>IF</string>
                    </property>
                   </item>
                   <item>
                    <property name="text">
                     <string>Absolute</string>
                    </property>
                   </item>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_7">
                   <property name="text">
                    <string>Frequency:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="1">
                  <widget class="SIUnitEdit" name="LOFrequency">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="0">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>IF1:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="SIUnitEdit" name="IF"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QGroupBox" name="groupBox_6">
                 <property name="title">
                  <string>Amplifier</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_13">
                  <item>
                   <widget class="QCheckBox" name="LOAmplifierEnable">
                    <property name="text">
                     <string>Enable</string>
                    </property>
                    <property name="checked">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_2">
                 <property name="title">
                  <string>Path</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_6">
                  <item>
                   <widget class="QRadioButton" name="LOInternal">
                    <property name="text">
                     <string>Internal</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">buttonGroup</string>
                    </attribute>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="LOExternal">
                    <property name="text">
                     <string>External</string>
                    </property>
                    <attribute name="buttonGroup">
                     <string notr="true">buttonGroup</string>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string>Aquisition</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QCheckBox" name="Port1Enable">
               <property name="text">
                <string>Port  Enable</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="RefEnable">
               <property name="text">
                <string>Reference Enable</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QFormLayout" name="formLayout_5">
               <item row="0" column="1">
                <widget class="QComboBox" name="portgain">
                 <item>
                  <property name="text">
                   <string>1V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>10V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>20V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>30V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>40V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>60V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>80V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>120V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>157V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>0.25V/V</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="refgain">
                 <item>
                  <property name="text">
                   <string>1V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>10V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>20V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>30V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>40V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>60V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>80V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>120V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>157V/V</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>0.25V/V</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="label_12">
                 <property name="text">
                  <string>Samples:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QSpinBox" name="Samples">
                 <property name="minimum">
                  <number>96</number>
                 </property>
                 <property name="maximum">
                  <number>16384</number>
                 </property>
                 <property name="singleStep">
                  <number>16</number>
                 </property>
                 <property name="value">
                  <number>16384</number>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_15">
                 <property name="text">
                  <string>Window:</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="cbWindow">
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Kaiser</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Hann</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Flat Top</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Reference PGA gain:</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Port PGA gain:</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_12">
       <property name="title">
        <string>Measurements</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QGroupBox" name="groupBox_16">
          <property name="title">
           <string>Port</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_19">
           <item>
            <layout class="QFormLayout" name="formLayout_9">
             <item row="0" column="0">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>ADC min:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="portmin"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>ADC max:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="portmax"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>Magnitude:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="portmag"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>Phase:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="portphase"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Referenced:</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="portreferenced"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_15">
          <property name="title">
           <string>Reference</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_18">
           <item>
            <layout class="QFormLayout" name="formLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>ADC min:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="refmin"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>ADC max:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="refmax"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>Magnitude:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="refmag"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Phase:</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="refphase"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SIUnitEdit</class>
   <extends>QLineEdit</extends>
   <header>CustomWidgets/siunitedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
